mpsl/CHANGELOG.rst

```diff
@@ -76,8 +72,8 @@ Changes
 
 * Removed ``_nrf52`` from the MPSL library file names.
 * Version numbers have been removed from the libraries.
-* It is no longer allowed to call :c:func:`mpsl_init` if MPSL is already initialized.
-* Clock configuration parameters for any stack that uses MPSL must be provided in :c:type:`mpsl_clock_lfclk_cfg_t` to :c:func:`mpsl_init`.
+* It is no longer allowed to call :cpp:func:`mpsl_init()` if MPSL is already initialized.
```

Comment:
This looks like it is reverting changes, looks like nrfxlib changes have not been updated in MPSL.

Reply:
I was not aware of that. Fixed.

Reply:
@rugeGerritsen Is this something that should be changed? Documentation updates in nrfxlib not being synced by to the original copy.